sympy/stats/joint_rv.py

```diff
@@ -332,7 +329,7 @@ def pdf(self, *x):
             count = len(expr.domain.args)
             x = Dummy('x', real=True, finite=True)
             syms = [Indexed(x, i) for i in count]
-            expr = expression.pdf(syms)
+            expr = expr.pdf(syms)
         return Lambda(syms, self.compute_pdf(expr, marginalise_out))(*x)
```

Comment:
since items were added to marginalise_out, it might be smart to make that canonical as `list(ordered(marginalise_out))`.